projects/components/src/select/select.component.ts

```diff
@@ -164,3 +176,8 @@ export class SelectComponent<V> implements AfterContentInit, OnChanges {
     return this.items.find(item => item.value === value);
   }
 }
+
+export const enum SelectTriggerDisplayMode {
+  MenuWithChevron = 'menu-with-chevron',
+  Button = 'button'
```

Comment:
Based on the screenshot, this second mode seems more like an Icon display rather than button - I'd expect button to have borders

Reply:
It could be text without the icon as well or icon + text. Do you have an alternate name suggestion? Label?

Reply:
Hmm - do we have a use case case for icon + text? That seems a bit strange to me. If we do have that, I'd make it a third mode - specifically because the icon mode that we need for that use case should still accept a label, but display it as the tooltip IMO.

Reply:
We currently don't have a use case for icon + text, but I was thinking it is a possibility we want to show just those two, without the chevron and borders.

Reply:
I actually misread what the text is - it's not just a descriptor/placeholder, but the selection. In the icon case, we don't want the current selection to show, do we? I thought we actually want to style differently when theres a selection.
No selections:
![image](https://user-images.githubusercontent.com/45047841/106479100-c7fecc80-645e-11eb-9ecd-28e8cbce0866.png)

Active selections (not sure if that border/background is meant to be there always, or if that's only when pressed)
![image](https://user-images.githubusercontent.com/45047841/106479191-e49b0480-645e-11eb-8670-b4c7d76e8111.png)

Reply:
In case of icon, we do not want to show the current selection.

My understanding is that the highlighting is for when the menu is active. That'll need a bit more work. Since the icon display is not an independent standalone item there, showing the circular highlighting etc needs some more changes.

Reply:
Now that I think of this, I think it might be better to have an icon only mode, with the `trigger-content` section itself separate, so that the checks/stylings are cleaner. What do you think @aaron-steinfeld ?

Reply:
I think that's what I was pushing for, depending on exactly what you mean. What I was thinking was two modes (for now) the existing one and an icon only one. The template for select could specify the trigger via an ngSwitch and switch between the correct template for the mode.

> My understanding is that the highlighting is for when the menu is active

I would think the blue should be when it's got a non-empty selection, and the blue with the border/background would be when it's pressed (i.e. the menu is active). We can confirm that.

Reply:
Yea. that makes more sense. discarding this.